src/views/Stopwatch.vue

```diff
@@ -17,22 +17,25 @@ div
   hr
 
   div.row
-    div.col-md-6
-      h3 Running
+    div.col-md-12
       div(v-if="runningTimers.length > 0")
+        h3 Running
         div(v-for="e in runningTimers")
           stopwatch-entry(:event="e", :bucket_id="bucket_id", :now="now",
             @delete="deleteTimer", @update="updateTimer")
           hr(style="margin: 0")
       div(v-else)
         span(style="color: #555") No stopwatch running
+        hr
 
-    div.col-md-6
-      h3 Stopped
-      div(v-for="e in stoppedTimers")
-        stopwatch-entry(:event="e", :bucket_id="bucket_id", :now="now",
-          @delete="deleteTimer", @update="updateTimer", @new="startTimer(e.data.label)")
-        hr(style="margin: 0")
+      div.mb-2
+
+      div(v-for="(v, k) in timersByDate")
+        h5.mt-4 {{ k }}
+        div(v-for="e in timersByDate[k]")
+          stopwatch-entry(:event="e", :bucket_id="bucket_id", :now="now",
+            @delete="deleteTimer", @update="updateTimer", @new="startTimer(e.data.label)")
```

Comment:
I'm not sure if I broke it or if you broke it but "deleteTimer" seems to be broken

Reply:
Fixed!